### PR TITLE
cookbook passing rubocop and foodcritic

### PR DESCRIPTION
**Pull request by codymalick**
GWM cookbook is passing rake style and rake lint.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -28,8 +28,8 @@
   action :create
 end
 
-no_clone = node.chef_environment == "vagrant" &&
-  ::File.directory?(::File.join(node['ganeti_webmgr']['path'], '.git'))
+# no_clone = node.chef_environment == 'vagrant' &&
+# ::File.directory?(::File.join(node['ganeti_webmgr']['path'], '.git'))
```
Comment (jldugger) [inline recipes/default.rb]: git can remember the past for us, no need to retain commented out code.

File: recipes/apache.rb [hunk 2]
```diff
@@ -32,11 +33,11 @@
...
   template_name = 'gwm_apache_vhost_https.conf.erb'
-  if gwm['apache']['server_port'] == 80
-    server_port = 443
-  else
-    server_port = gwm['apache']['server_port']
-  end
+  server_port = if gwm['apache']['server_port'] == 80
+                  443
+                else
+                  gwm['apache']['server_port']
```
Comment (jldugger) [inline recipes/apache.rb]: The refactor looks good, but I have no idea why this code does what it does :confused:

File: recipes/default.rb [hunk 3]
```diff
@@ -43,37 +43,38 @@
...
+#             when 'psycopg2', 'postgresql_psycopg2'
+#               'postgres'
+#             when 'sqlite3'
+#               'sqlite'
+#             else
+#               log "node['ganeti_webmgr']['database']['engine'] needs to
+#                 be set!" do
+#                 level :fatal
+#               end
+#             end
```
Comment (jldugger) [inline recipes/default.rb]: um.... what?

File: recipes/default.rb (hunk 1 above)
Comment (codymalick) [inline recipes/default.rb]: I actually un-commented this later. Rubocop was declaring it 'useless assignments' when in fact it was needed. I can adjust this.

File: recipes/apache.rb (hunk 2 above)
Comment (codymalick) [inline recipes/apache.rb]: I assume it has something to do with https (port 443), but I'm not sure.

Comment (fahlmant) [inline recipes/apache.rb]: Did rubocop want this refactor?

File: recipes/default.rb [hunk 4]
```diff
@@ -43,37 +43,38 @@
...
 
 # the install dir *is* the virtualenv
 install_dir = node['ganeti_webmgr']['install_dir']
 
 # use setup.sh to install GWM
-execute "install_gwm" do
-  command "./scripts/setup.sh -D #{db_driver} -d #{install_dir}"
+execute 'install_gwm' do
+  command './scripts/setup.sh -D #{db_driver} -d #{install_dir}'
```
Comment (fahlmant) [inline recipes/default.rb]: Change this line back to " "

File: recipes/apache.rb (hunk 2 above)
Comment (codymalick) [inline recipes/apache.rb]: Yes, the current version wants one assignment from the if statement rather than two assignments nested in an if statement.

Comment (jldugger) [inline recipes/apache.rb]: Yea, it'll ask you to hoist assignments like that. It's fine.

File: recipes/default.rb (hunk 3 above)
Comment (codymalick) [inline recipes/default.rb]: Commented out code was fixed by Taylor's commit

**Pull request merged**